src/psm/README.md

```diff
@@ -40,8 +41,9 @@ Options description:
 - ``enable_em``: (optional) is the flag to report current per power grid segment.
 - ``outfile``: (optional) filename specified per-instance voltage written into file.
 - ``em_outfile``: (optional) filename to write out the per segment current values into a file, can be specified only if enable_em is flag exists.
-- ``voltage``: Sets the voltage on a specific net. If this command is not run, the voltage value is obtained from operating conditions in the liberty.
-- ``node_pitch``: This value can be specfied by the user in um to determine the pitch of the PDN nodes in M1 layer.
+- ``voltage``: (optional) Sets the voltage on a specific net. If this command is not run, the voltage value is obtained from operating conditions in the liberty.
+- ``node_density``: (optional)  This value can be specfied by the user in um to determine the pitch of the PDN nodes in M1 layer.
+- ``node_density_factor``: (optional) Integer value factor which is multiepled by standard cell heigh to set node denisty.
```

Comment:
Are these mutually exclusive?

Reply:
Yes. Either the user can directly set the node density or set it as a factor of the std. cell height.

Reply:
Then there should be a check in tcl that gives an error if both are used and some clarification in the documentation.